src/core/libmaven/mzSample.cpp

```diff
@@ -963,10 +968,24 @@ void mzSample::parseMzXMLScan(const xml_node &scan, const int& scannum)
 		scanpolarity = getPolarityFromfilterLine(filterLine);
 	}
 
-	//TODO: What is this for this is zero
-	precursorMz = string2float(string(scan.child_value("precursorMz")));
-
-	// string b64String; naman Unused variable: b64String
+    xml_node precursor = mzxml_scan_node.child("precursorMz");
```

Comment:
The variable `mzxml_scan_node` is not known in this scope. This is one of the issues that fail to compile.

Reply:
Same as above. Pushing the changes.